src/main/java/frc/robot/Constants.java

```diff
@@ -24,7 +27,7 @@ public final class Constants {
         final public static int[] rightDeviceID = new int[] { 4, 5, 6 };
 
         //Intake CAN IDs
-        final public static int[] intakeDeviceID = new int[] { 8, 9, 10 }; 
+        final public static int[] intakeDeviceID = new int[] { 8, 9, 7 }; 
                                                                        
         //Shooter CAN IDs
```

Comment:
The values in the array should be constants that you define earlier in the file so you can then easily refer to them as:

public static final int FRONT_LEFT_MOTOR = 4;
public static final int MID_LEFT_MOTOR = 5;
public static final int REAR_LEFT_MOTOR = 6;

final public static int[] leftDeviceID = new int[] { FRONT_LEFT_MOTOR, MID_LEFT_MOTOR, REAR_LEFT_MOTOR };

Alternately you can create nested classes to organize the constants in the code:

public class Constants {
...
    public static final class DriveConstants {
        public static final int FRONT_LEFT_MOTOR = 4;
        public static final int MID_LEFT_MOTOR = 5;
        public static final int REAR_LEFT_MOTOR = 6;

which you reference using the full class paths.  For example, Constants.DriveConstants.FRONT_LEFT_MOTOR.